src/core/tail.js

```
@@ -0,0 +1,79 @@
+    const num = raw !== undefined ? parseInt(raw.replace(/^[+-]/, ''), 10) : 10;
+    const lines = input.split('\n');
+    if (input.endsWith('\n')) lines.pop();
+    let selected;
+    if (raw === undefined || sign === null) {
+      selected = num >= lines.length ? lines : lines.slice(-num);
+    } else if (sign === '+') {
+      selected = num > 0 ? lines.slice(num - 1) : lines;
+    } else if (sign === '-') {
+      selected = num < lines.length ? lines.slice(num) : [];
+    }
+    if (selected.length > 0) stdout = selected.join('\n') + '\n';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)